src/pane.coffee

```diff
@@ -577,13 +577,26 @@ class Pane extends Model
     else
       return true
 
+    saveError = (error) =>
+      if error
+        chosen = @applicationDelegate.confirm
```

Comment:
I'd like to see this unified with the nearly duplicate code starting on line 593.